tap/jdg-tap-merge-confict

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+
+if ARGV[0].nil?
+  $stderr.puts "Usage: #{File.basename $0} <file>"
+  exit 1
+end
+
+if not system "which grep >/dev/null 2>&1"
```

Comment:
And also name the tool jdg-tap-merge-conf**l**ict? :)

Reply:
> Maybe this could use instead https://ruby-doc.org/stdlib-2.0.0/libdoc/mkmf/rdoc/MakeMakefile.html#method-i-find_executable ? I see which is being used in other scripts, but that might need fixing as there's been attempts to deprecate that program.

Using mkmf is quite a burden, since generates a mkmf.log file by default on execution, needs ruby-dev...